modules/common/package.json

```diff
@@ -17,13 +17,13 @@
     "node": ">=14"
```

Comment:
Should we unify the required node version?

Reply:
good idea. done